docs/SRS/SRS.tex

```diff
@@ -150,1121 +150,106 @@ \section*{Revision History}
-		\rref{R_timeMeltEnd}   &             & X           &               &               &                           &                    &                     \\
-		\hline
-	\end{tabular}
-	\caption{Traceability Matrix Showing the Connections Between Requirements and Instance Models}
-	\label{Table:R_trace}
-\end{table}
-
-The purpose of the traceability graphs is also to provide easy references on what has to be
-additionally modified if a certain component is changed. The arrows in the graphs represent
-dependencies. The component at the tail of an arrow is depended on by the component at the head of
-that arrow. Therefore, if a component is changed, the components that it points to should also be
-changed. Figure~\ref{Fig_ATrace} shows the dependencies of theoretical models, general definitions,
-data definitions, instance models, likely changes, and assumptions on each other.
-Figure~\ref{Fig_RTrace} shows the dependencies of instance models, requirements, and data
-constraints on each other.
-
-% \begin{figure}[h!]
-% 	\begin{center}
-% 		%\rotatebox{-90}
-% 		{
-% 			\includegraphics[width=\textwidth]{ATrace.png}
-% 		}
-% 		\caption{\label{Fig_ATrace} Traceability Matrix Showing the Connections Between Items of Different Sections}
-% 	\end{center}
-% \end{figure}
-
-% \begin{figure}[h!]
-% 	\begin{center}
-% 		%\rotatebox{-90}
-% 		{
-% 			\includegraphics[width=0.7\textwidth]{RTrace.png}
-% 		}
-% 		\caption{\label{Fig_RTrace} Traceability Matrix Showing the Connections Between Requirements, Instance Models, and Data Constraints}
-% 	\end{center}
-% \end{figure}
-
-\section{Development Plan}
-
-\plt{This section is optional.  It is used to explain the plan for developing
-	the software.  In particular, this section gives a list of the order in which
-	the requirements will be implemented.  In the context of a course  this is
-	where you can indicate which requirements will be implemented as part of the
-	course, and which will be ``faked'' as future work.  This section can be
-	organized as a prioritized list of requirements, or it could should the
-	requirements that will be implemented for ``phase 1'', ``phase 2'', etc.}
-
-\section{Values of Auxiliary Constants}
-
-\plt{Show the values of the symbolic parameters introduced in the report.}
-
-\plt{The definition of the requirements will likely call for SYMBOLIC\_CONSTANTS.
-	Their values are defined in this section for easy maintenance.}
-
-\plt{The value of FRACTION, for the Maintainability NFR would be given here.}
+This section is not in the original Volere template, but health and safety are issues that should
+be considered for every engineering project.
 
-\newpage
+\section{Project Issues}
+
+\subsection{Open Issues}
+
+\subsection{Off-the-Shelf Solutions}
+
+\subsection{New Problems}
+
+\subsection{Tasks}
 
-\bibliographystyle {plainnat}
-\bibliography {../../refs/References}
```

Comment:
Use the existing `refs/References.bib` instead of making a new `SRS.bib`.

Reply:
Updated